modules/40_techpol/NDC/bounds.gms

```diff
@@ -31,8 +26,6 @@ if(cm_nucscen eq 5,
   vm_cap.lo(t,regi_nucscen,"tnrs","1")$(t.val gt 2020) = 0;
 );
```

Comment:
why do we have this only in NDC and not in the other realizations of 40_techpol? And isn't this redundant with the bound in bounds.gms which applies starting 2015?
https://github.com/LaviniaBaumstark/remind/blob/69cb5f61eafc3a5bab0f25dfe5e4afb27934a7af/core/bounds.gms#L222

Reply:
I assume that this is specific for some national plans and the other realizations are not as up to date? 
In core/bounds.gms it is only active for nucscen=5

Reply:
here in NDC/bounds it is also only active for nucscen=5, right?

Reply:
hm, right, I will also delete this part...